src/test-run/index.js

```diff
@@ -1,4 +1,4 @@
-import { pull as remove } from 'lodash';
+import { pull, remove, chain } from 'lodash';
```

Comment:
I don't really like `pull`, because here we definitely have two versions of `remove`, but since its `lodash` naming let it be so